### PR TITLE
[noticket] Add options `documentRoot` and `defaultMatcherPaths`

### PR DESCRIPTION
**Pull request by jinnoflife**
- Add example for sw5 setup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,6 +73,23 @@ MySQL search.
...
 ```
 
+### kellerkinder.documentRoot
+Changes the default document root (`public`) of caddy to the specified value
+
+*_Example for the current folder_*
+```
+kellerkinder.documentRoot = ".";
+```
+### kellerkinder.defaultMatcherPaths
```
Comment (kleinmann) [inline README.md:83]: Maybe more along the lines of `staticFilePaths` to clarify what those are expected to be and hint at how caddy uses them?

File: examples/sw5/devenv.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+inputs:
+  kellerkinder:
+    url: path:/Users/awink/Code/k10r/devenv/devenv-shopware
```
Comment (jochenmanz) [inline examples/sw5/devenv.yaml:3]: Hier sollten wir den Pfad zum Github verwenden.

File: README.md (hunk 1 above)
Comment (jinnoflife) [reply README.md]: Good point - was adjusted

File: examples/sw5/devenv.yaml (hunk 2 above)
Comment (jinnoflife) [reply examples/sw5/devenv.yaml]: Aye, was changed

File: devenv.nix [hunk 3]
```diff
@@ -17,7 +17,8 @@ let
     ''}
     display_errors = On
     display_startup_errors = true
-    error_reporting = E_ALL
+    error_reporting = E_ALL & ~E_NOTICE & ~E_DEPRECATED
+    ;error_reporting = E_ALL
```
Comment (jochenmanz) [inline devenv.nix:21]: Is this change intentional?

Comment (jinnoflife) [reply devenv.nix]: Nope, removed that

**Review by jochenmanz — APPROVED** [review]
